src/components/SingleDatePicker.jsx

```diff
@@ -404,6 +404,7 @@ class SingleDatePicker extends React.Component {
           renderCalendarInfo={renderCalendarInfo}
           isFocused={isDayPickerFocused}
           showKeyboardShortcuts={showKeyboardShortcuts}
+          onBlur={this.onDayPickerBlur}
```

Comment:
This was another important miss I found in functionality between the DateRangePicker but not the SingleDatePicker.  This was the missing piece to the focus returning back to the DateInput when that was where the keyboard shortcuts panel was triggered to open.